webapp/channels/src/root.html

```diff
@@ -19,68 +19,13 @@
     <link rel="icon" type="image/png" href="/static/images/favicon/favicon-default-64x64.png" sizes="64x64">
     <link rel="icon" type="image/png" href="/static/images/favicon/favicon-default-96x96.png" sizes="96x96">
 
-    <!-- CSS Should always go first -->
-    <link rel='stylesheet' class='code_theme'>
```

Comment:
Not used anymore https://github.com/mattermost/mattermost-webapp/commit/ad2843c11eb568e0c79baeb48bed9a4249c6d12b